release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -849,6 +849,24 @@ Helm-based Operator enhancements include:
 {product-title} now properly supports the `terminationGracePeriodSeconds`
 parameter with the CRI-O container runtime.
 
+[discrete]
+[id="ocp-4-5-readyz-support"]
+==== /readyz endpoint support for API server health probe
+
+The `/readyz` endpoint for API server health checking is now supported and
+required for {product-title} clusters.
```

Comment:
"now supported" is not wrong wording IMO. It's supported longer. The point is that it is to be used to get a supportable cluster. From kube-apiserver point of view it has always been required. Our UPI docs were lacking.

Reply:
@sttts I see. Just so we're on the same page, I'd like to clarify the `/readyz` status throughout OCP 4.x.

Is there anything new for `/readyz` configs in OCP 4.5? Or, starting in OCP 4.5, we only support clusters with `/readyz` configured? I'm trying to find what's **new** that we're highlighting about this.

Or are we just trying to make a point that our docs were not sufficient on this topic, and that it's doc'd now? I'll move it out of the *New tech changes* section if there isn't really anything new about it.

Reply:
For whatever internal reasons the UPI docs were not reviewed and the missing reference to `/readyz` went unnoticed. Our IPI config in the installer always used `/readyz` or before that `/healthz` (maybe in 4.1).

Maybe a good way out of this:

- UPI installation with the proper use of `/readyz` will suffer from API outages (length depends on the LB in use, and on the time the kube-apiserver needs to restart. I.e. this length in be in the area of tens of seconds to few minutes) whenever the kube-apiserver restarts within the cluster. This can happen for many reason, e.g. config changes, certificate updates or master node reboots.
- Cluster installed before 4.5 have to be reconfigured to use proper `/readyz` configuration.
- New cluster must use this configuration from the beginning.

Reply:
@sttts thanks for that info. I've rewritten the release note based on your feedback. Can you confirm?

Updated preview [here](http://file.rdu.redhat.com/~choag/080520/rn-4.5-readyz/release_notes/ocp-4-5-release-notes.html#ocp-4-5-notable-technical-changes).